### PR TITLE
[#4101] Disable "Show form" button if form not active

### LINKED ISSUE #4101: If form is not active do not show "show form" button in list/detail view

### PR DESCRIPTION
**Pull request by Viicos**
Fixes #4101

Using `pointer-events: none;` prevents `title` from showing on hoover, and using a top level `span` element with a `title` didn't help either :/

### CODE REVIEW

**Review by vaszig — COMMENTED** [review]
Is it a problem on my end that I keep on seeing a link even when the form is inactive?
If I click it I get a 404.

**Comment by Viicos** [comment]
> Is it a problem on my end that I keep on seeing a link even when the form is inactive?
> If I click it I get a 404.

Hum probably you need to run `npm run start/build` first?

**Comment by vaszig** [comment]
> > Is it a problem on my end that I keep on seeing a link even when the form is inactive?
> > If I click it I get a 404.
> 
> Hum probably you need to run `npm run start/build` first?

As we discussed in the office I was talking about the wrong button!It's' working for the `form_change` page but we have to do the same for the list page of the forms.

**Comment by Viicos** [comment]
I'll first need to know if this worth the effort, as the list view list is defined as a Django `list_display` field, making it hard to customize depending on the `active` value.

File: src/openforms/forms/admin/form.py [hunk 1]
```diff
@@ -235,12 +236,14 @@ def get_queryset(self, request):
         )
 
     @admin.display(description=_("Actions"))
-    def get_object_actions(self, obj) -> str:
-        links = ((obj.get_absolute_url(), _("Show form")),)
+    def get_object_actions(self, obj: Form) -> str:
+        links: list[tuple[str, StrOrPromise]] = []
+        if obj.active:
+            links.append((obj.get_absolute_url(), _("Show form")))
```
Comment (Viicos) [inline src/openforms/forms/admin/form.py:242]: The link could also be made disabled: I made a draft implementation but this adds complexity to the code so let me know if it is necessary.

Comment (sergei-maertens) [reply src/openforms/forms/admin/form.py:242]: this is fine!

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**